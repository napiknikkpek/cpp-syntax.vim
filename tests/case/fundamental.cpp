int i;
void f(bool, float f, char32_t c);
template <int N>
unsigned short foo();
