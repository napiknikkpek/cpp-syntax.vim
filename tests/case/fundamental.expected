zzz.x.
zzzz.f.zzzz..zzzzz.x..zzzzzzzz.x..
aaaaaaaa..zzz.x.
zzzzzzzz.zzzzz.fff...
